projects/opendr_ws/src/opendr_perception/scripts/facial_emotion_estimation_node.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import torch
+import numpy as np
+import cv2
+from torchvision import transforms
+import PIL
+
+import rospy
+from std_msgs.msg import String
+from vision_msgs.msg import ObjectHypothesis
+from sensor_msgs.msg import Image as ROS_Image
+from opendr_bridge import ROSBridge
+
+from opendr.engine.constants import OPENDR_SERVER_URL
```

Comment:
Sorry Negar i missed this when i updated the node. This import is unused, could you remove it when you get the chance? I am not doing it myself to not mess up your branch.

```suggestion
```